frontend/public/components/notification-drawer.tsx

```diff
@@ -38,22 +40,33 @@ import {
 import { getSortedUpdates } from './modals/cluster-update-modal';
 import { usePrevious } from '@console/metal3-plugin/src/hooks';
 
-const emptyState = (toggleExpanded: () => void) => (
+const errorState = (
   <EmptyState variant={EmptyStateVariant.full} className="co-status-card__alerts-msg">
-    <Title headingLevel="h5" size="lg">
-      No critical alerts
-    </Title>
-    <EmptyStateBody>
-      There are currently no critical alerts firing. There may be firing alerts of other severities
-      or silenced critical alerts however.
-    </EmptyStateBody>
-    <EmptyStateSecondaryActions>
-      <Link to="/monitoring/alerts" onClick={toggleExpanded}>
-        View all alerts
-      </Link>
-    </EmptyStateSecondaryActions>
+    <EmptyStateIcon className="co-status-card__alerts-icon" icon={UnknownIcon} />
+    <EmptyStateBody>Alerts could not be loaded.</EmptyStateBody>
   </EmptyState>
 );
+
+const emptyState = (toggleExpanded: () => void, hasErrors: boolean) =>
+  hasErrors ? (
+    errorState
```

Comment:
I'd either assign both possibilities to different variables or inline both. It's odd to assign one to a var and inline the other.